### ISSUE TITLE
[BUG] Crashing on the start

### ISSUE THREAD

**Issue by Unlky** [DarkKronicle/AdvancedChatFilters#11]
**Describe the bug**
Crash on the start

**Minecraft Info**
Minecraft information:
- AdvancedChatCore 1.18.2 1.4.4
- AdvancedChatFilters 1.18.2 1.2.1
- AdvancedChatHUD 1.18.2 1.2.3
- malilib 1.18.2 0.12.0

**Expected behavior**
Idk what is happening that much but its crashing when i try to load the game. Here is my log:

[latest.log](https://github.com/DarkKronicle/AdvancedChatFilters/files/8211220/latest.log)

**Comment by Unlky** [comment]
I think its problem of AdvancedChatFilters 1.18.2 1.2.1 other 3 of them is working without it

**Comment by Xaver106** [comment]
I can confirm that this is a problem of AdvancedChatFilters 1.18.2 1.2.1

Loaded Mods:

- AdvancedChatCore-1.18.2-1.4.4
- AdvancedChatFilters-1.18.2-1.2.1
- fabric-api-0.47.10+1.18.2
- malilib-fabric-1.18.2-0.12.0

Running with only AdvancedChatCore and malilib-fabric works.

Latest log:
```[19:52:14] [main/INFO]: Loading Minecraft 1.18.2 with Fabric Loader 0.13.3
[19:52:15] [main/INFO]: Loading 59 mods:
	- advancedchatcore 1.18.2-1.4.4
	- advancedchatfilters 1.18.2-1.2.1
	- com_electronwill_night-config_core 3.6.5 via advancedchatcore
	- com_electronwill_night-config_toml 3.6.5 via advancedchatcore
	- com_github_darkkronicle_konstruct_addons 2.0.2-build1 via advancedchatcore
	- com_github_darkkronicle_konstruct_core 2.0.2-build1 via advancedchatcore
	- fabric 0.47.10+1.18.2
	- fabric-api-base 0.4.3+d7c144a860 via fabric
	- fabric-api-lookup-api-v1 1.5.6+d7c144a860 via fabric
	- fabric-biome-api-v1 8.0.2+801ec85b60 via fabric
	- fabric-blockrenderlayer-v1 1.1.11+3ac43d9560 via fabric
	- fabric-command-api-v1 1.1.8+d7c144a860 via fabric
	- fabric-commands-v0 0.2.7+b4f4f6cd60 via fabric
	- fabric-containers-v0 0.1.20+d7c144a860 via fabric
	- fabric-content-registries-v0 3.0.1+d82b939260 via fabric
	- fabric-crash-report-info-v1 0.1.10+3ac43d9560 via fabric
	- fabric-dimensions-v1 2.1.16+2540745460 via fabric
	- fabric-entity-events-v1 1.4.7+d7c144a860 via fabric
	- fabric-events-interaction-v0 0.4.18+d7c144a860 via fabric
	- fabric-events-lifecycle-v0 0.2.12+d7c144a860 via fabric
	- fabric-game-rule-api-v1 1.0.12+d7c144a860 via fabric
	- fabric-item-api-v1 1.3.2+691a79b560 via fabric
	- fabric-item-groups-v0 0.3.9+3ac43d9560 via fabric
	- fabric-key-binding-api-v1 1.0.11+54e5b2ec60 via fabric
	- fabric-keybindings-v0 0.2.9+b4f4f6cd60 via fabric
	- fabric-lifecycle-events-v1 2.0.1+2540745460 via fabric
	- fabric-loot-tables-v1 1.0.10+d7c144a860 via fabric
	- fabric-mining-level-api-v1 2.0.3+d1027f7d60 via fabric
	- fabric-models-v0 0.3.5+d7c144a860 via fabric
	- fabric-networking-api-v1 1.0.20+d882b91560 via fabric
	- fabric-networking-v0 0.3.7+b4f4f6cd60 via fabric
	- fabric-object-builder-api-v1 2.0.2+34b2a0ce60 via fabric
	- fabric-particles-v1 0.2.11+526dc1ac60 via fabric
	- fabric-registry-sync-v0 0.9.7+0d9ab37260 via fabric
	- fabric-renderer-api-v1 0.4.12+d882b91560 via fabric
	- fabric-renderer-indigo 0.4.16+d8c7b9ae60 via fabric
	- fabric-renderer-registries-v1 3.2.10+b4f4f6cd60 via fabric
	- fabric-rendering-data-attachment-v1 0.3.6+d7c144a860 via fabric
	- fabric-rendering-fluids-v1 2.0.1+54e5b2ec60 via fabric
	- fabric-rendering-v0 1.1.12+b4f4f6cd60 via fabric
	- fabric-rendering-v1 1.10.6+54e5b2ec60 via fabric
	- fabric-resource-conditions-api-v1 2.0.1+d1027f7d60 via fabric
	- fabric-resource-loader-v0 0.4.17+801ec85b60 via fabric
	- fabric-screen-api-v1 1.0.9+d882b91560 via fabric
	- fabric-screen-handler-api-v1 1.1.13+d7c144a860 via fabric
	- fabric-textures-v0 1.0.11+3ac43d9560 via fabric
	- fabric-transfer-api-v1 1.5.13+cca23f9360 via fabric
	- fabricloader 0.13.3
	- io_github_maowimpl_owo 2.0.0 via advancedchatcore
	- java 17
	- kommandlib 0.1.0-1.18.1 via advancedchatcore
	- malilib 0.12.0
	- minecraft 1.18.2
	- org_apache_commons_commons-csv 1.8 via advancedchatcore
	- org_javadelight_delight-nashorn-sandbox 0.2.5 via advancedchatfilters
	- org_mariuszgromada_math_mathparser_org-mxparser 4.4.2 via advancedchatcore
	- org_openjdk_nashorn_nashorn-core 15.3 via advancedchatfilters
	- org_slf4j_slf4j-api 1.7.25 via advancedchatfilters
	- org_webjars_bower_js-beautify 1.9.0 via advancedchatfilters
[19:52:15] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/xaver/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.11.2+mixin.0.8.5/sponge-mixin-0.11.2+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[19:52:15] [main/INFO]: Compatibility level set to JAVA_16
[19:52:15] [main/INFO]: Compatibility level set to JAVA_17
[19:52:16] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.LinkageError: loader constraint violation: loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @266374ef wants to load interface org.slf4j.Logger. A different interface with the same name was previously loaded by 'app'. (org.slf4j.Logger is in unnamed module of loader 'app')
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:418) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.13.3.jar:?]
Caused by: java.lang.LinkageError: loader constraint violation: loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @266374ef wants to load interface org.slf4j.Logger. A different interface with the same name was previously loaded by 'app'. (org.slf4j.Logger is in unnamed module of loader 'app')
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1012) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:223) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:174) ~[fabric-loader-0.13.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.13.3.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at org.slf4j.LoggerFactory.<clinit>(LoggerFactory.java:86) ~[org_slf4j_slf4j-api-1.7.25-3f9f6f3373c2a24e.jar:?]
	at net.minecraft.class_2370.<clinit>(class_2370.java:38) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:130) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:49) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:153) ~[fabric-loader-0.13.3-1.18.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416) ~[fabric-loader-0.13.3.jar:?]
	... 2 more
```

**Comment by Keeg-Turner** [comment]
Might want to check this out: https://github.com/FabricMC/fabric/issues/2064
To paraphrase, the issue seems to be that AdvancedChatFilters is bundling with `sfl4j`, which is now included in Minecraft by default as of `1.18.2`.

**Comment by DarkKronicle** [comment]
Fixed
